common/lfdemod.c

```diff
@@ -1067,6 +1089,23 @@ void psk1TOpsk2(uint8_t *BitStream, size_t size)
 	return;
 }
 
+// by marshmellow
+// convert psk2 demod to psk1 demod
+// from only transition waves are 1s to phase shifts change bit
+void psk2TOpsk1(uint8_t *BitStream, size_t size)
```

Comment:
I've been looking at this a while now. I don't understand -- if the BitStream[i] differs from the previous one (say, was 1, is now 0), phase becomes 0, and Bitstream[i] is thus set to 0. Does not this leave BitStream the same as it was before ? What am I missing ? Sorry.